### ISSUE TITLE
Idea: Tar and Torch

### ISSUE THREAD

**Issue by WarttHog** [StoneAgeFactorio/StoneAgeFactorio#14]
Another possible early resource would be "tar", which could be necessary to waterproof the pump and wooden pipe.  Also, tar on a stick could be a torch, an early lamp!  It could pollute and perhaps give less light than a normal lamp, but it's probably okay for it to burn forever.

**Comment by RomkeVdMeulen** [comment]
I like it! Where would the player get tar from though?

**Comment by WarttHog** [comment]
Ideally by dunking their stick or pipe or whatever in an oil patch, but I have no idea how you'd pull that off.

**Comment by RomkeVdMeulen** [comment]
Nice! That could work. It'd require an oil patch somewhere close to the starting area though, and:
> Oil and uranium were never intended to be in the starting area
-- [FFF 258](https://www.factorio.com/blog/post/fff-258)

I could spawn one manually, but that'd mean the player has a way to get plastic without leaving the starting area, so that affects vanilla gameplay.

On the other hand, the player could go out exploring for an oil patch, provided the biter settings aren't too high. I doubt many players would take the effort for something as simple as a light though.

Maybe another way. You could make one out of sticks and vines, but those materials are already being used for a number of recipes so that feels repetitive.

I'll give it some thought. I definitely want torches in the mod, just have to find the right way.

**Comment by TomGault** [comment]
Yeah true.  Adding it to the pump recipe makes it mandatory but then it's also late in the mod's tech tree.  I kind of like the idea of a dangerous, mandatory, stealth adventure early in the game!  Maybe you could craft a bucket, use it to collect tar from an oil well once then it would have infinite tar for all your Stone Age needs.

**Comment by RomkeVdMeulen** [comment]
Stealth mission, that sounds fun. But if it only unlocks the torch, which you can do without, then I image most players won't bother. On the other hand, I'm not sure making a stealth mission mandatory is advisable either.

Perhaps if, like you suggested in #17, we make some primitive armor and weapons so you're more likely to survive the trip, we could justify making the tar mandatory...

**Comment by WarttHog** [comment]
Whoops!  Accidentally switched to my public-facing work GitHub account!  :blush:   Here's the comment Romke was replying to above:

> Yeah true. Adding it to the pump recipe makes it mandatory but then it's also late in the mod's tech tree. I kind of like the idea of a dangerous, mandatory, stealth adventure early in the game! Maybe you could craft a bucket, use it to collect tar from an oil well once then it would have infinite tar for all your Stone Age needs.

Torches and pipes/pump, but yeah I totally see your point. :+1:

**Comment by RomkeVdMeulen** [comment]
Or, biters drop intestines that can be rendered down to tar? Requiring the player to fight a couple of biters at least, using the armor suggested in #17?

**Comment by RomkeVdMeulen** [comment]
How's this? Since tar is proving so difficult, how about I add torches for now as made from sticks and vines?

**Comment by RomkeVdMeulen** [comment]
Ow man, turns out it's impossible to make a kind of lamp that doesn't run on electric power. It's possible to make a workaround, where a torch is made as a hidden boiler and generator. Klonan made a [mod that does exactly that](https://mods.factorio.com/mod/Torches). It takes a lot of work though, to make sure all three buildings are placed and removed (or destroyed) as a set.

I don't think I want to get into this, at least not for the first version. I'll move this to long term.

**Comment by RomkeVdMeulen** [comment]
See #23: support for buildings that don't use power is coming in 0.17. Until then, this will have to be on hold.
